applications/ALSAFullDuplex.C

```diff
@@ -114,6 +118,8 @@ int main(int argc, char *argv[]) {
       ;
   if (op.getArg<string>("C", argc, argv, deviceNameIn, i=0)!=0)
       ;
+  if (op.getArg<int>("R", argc, argv, fs, i=0)!=0)
```

Comment:
Adhere to aplay type formatting use aplay -h to see. There sample rate is the lower case R